CONTRIBUTING.md

```diff
@@ -41,65 +43,171 @@ These are the relevant environment variables:
+Create the `kubewarden` namespace
 
 ```console
 kubectl create ns kubewarden
 ```
 
-### Running
+Now, as we are using `telepresence` to intercept the intra cluster
+communication, we need to create the controller deployment and service that
+`telepresence` will intercept. For this, install the Kuberwarden stack
+following the steps described quickstart guide. In other words, install
+Kubewarden controller helm chart. However, a change is required. The controller
+deployment must allow root users. This is disable by default and it is required
+because `telepresence` will add a init container in the deployment which need
+root access.
+
+```console
+# Remember to install the requirements needed. See the quickstart guide for more information
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
+kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+helm install --wait -n kubewarden  kubewarden-controller kubewarden/kubewarden-controller
+```
+
+
+
+Now, install `telepresence` in the cluster and connect it to the `kubewarden` namespace
+
+```console
+telepresence helm install
+telepresence connect -n kubewarden
+```
+
+At this point you should be able to reach workloads running in the cluster. To validate
+this use a `curl` command:
+
+```console
+curl -XGET --insecure https://kubewarden-controller-webhook-service.kubewarden.svc.cluster.local/mutate-policies-kubewarden-io-v1-admissionpolicy
+```
+
+In order to avoid two controllers trying to reconcile the same resources, it's necessary to
+remove the deployment installed by the `kubewarden-controller` helm chart and deploy a dummy
+deployment just to allow `telepresence` to intercept the communications.
+
+```console
+kubectl delete deployment -n kubewarden kubewarden-controller
+kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/instance: kubewarden-controller
+    app.kubernetes.io/name: kubewarden-controller
+  name: kubewarden-controller
+  namespace: kubewarden
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: kubewarden-controller
+      app.kubernetes.io/name: kubewarden-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: kubewarden-controller
+        app.kubernetes.io/name: kubewarden-controller
+    spec:
+      containers:
+      - name: echo-server
+        image: gcr.io/google-containers/echoserver:1.8
+        ports:
+        - containerPort: 9443
+      serviceAccount: kubewarden-controller
+      serviceAccountName: kubewarden-controller
+  replicas: 1
+EOF
+```
+
+> Note: if you want to use some kind o NetworkPolicy to block the original controller traffic
+> you may need to have a multi node cluster. Because in a single node cluster the traffic 
+> policies will not be applied. Because the traffic will not leave the node and never the
+> checked against the network policies
```

Comment:
```suggestion
> Note: if you want to use some kind of NetworkPolicy to block the original controller traffic
> you may need to have a multi node cluster. In a single node cluster the traffic 
> policies will not be applied. Because the traffic will not leave the node and will never be
> checked against the network policies
```